### PR TITLE
Image placeholder

### LINKED ISSUE #302: Missing images on Articles (404)
When visiting: http://www.tcdl.io.s3-website-eu-west-1.amazonaws.com/isearch/0.18/index.html and searching for `Spanien` we see articles for Barcelona, a few of these have "broken" images (`404`)
isearch-missing-article-images-404

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
fixes #302 #304 #305 
* fixes the layout for the mobile view on android when searchbar is clicked
* adds placeholder image for when there isn't an image available

### CODE REVIEW

File: lib/article-tile/index.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React, { Component, PropTypes } from 'react';
 require('./style.css');
 import FadeImage from '../fade-image';
+import defaultImage from '../../src/constants/default-image';
```
Comment (nelsonic) [inline lib/article-tile/index.js]: any reason this default image is not simply uploaded to CDN (e.g: S3) _once_?
does the `import` an image add it to the `bundle.js` or does it get shipped to S3 and loaded using "isomorphic fetch" ...?

File: src/constants/default-image.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const defaultImage = 'http://www.tcdl.io.s3-website-eu-west-1.amazonaws.com/isearch/images/thomascook_fallback-image.jpg';
```
Comment (nelsonic) [inline src/constants/default-image.js]: is there an `https` version of this so we don't get a [`mixed content warning`](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#Warnings_in_Web_Console) ?

Comment (nelsonic) [inline src/constants/default-image.js]: Try this: https://isearch-ui.s3.amazonaws.com/thomascook_fallback-image.jpg

Comment (nelsonic) [inline src/constants/default-image.js]: @jackcarlisle would you mind updating it to: https://isearch-ui.s3.amazonaws.com/thomascook_fallback-image.jpg ?

Comment (jackcarlisle) [inline src/constants/default-image.js]: sure!

**Comment by nelsonic** [comment]
thanks for making those changes. 👍

**Pull request merged**